src/Renderer/HtmlRenderer.php

```diff
@@ -491,16 +493,33 @@ public function createServerInformationLink(ServerRequestInterface $request): st
     }
 
     /**
-     * Returns the name of the throwable instance.
+     * Returns string representation of the throwable name.
      *
-     * @return string The name of the throwable instance.
+     * @param Throwable $throwable The throwable.
+     *
+     * @return string The throwable name.
      */
     public function getThrowableName(Throwable $throwable): string
     {
         $name = $throwable::class;
 
         if ($throwable instanceof FriendlyExceptionInterface) {
             $name = $throwable->getName() . ' (' . $name . ')';
+        } else {
+            // Check if the exception class has FriendlyException attribute
+            try {
+                $reflectionClass = new ReflectionClass($throwable);
+                if (class_exists('Yiisoft\FriendlyException\Attribute\FriendlyException')) {
+                    $attributes = $reflectionClass->getAttributes('Yiisoft\FriendlyException\Attribute\FriendlyException');
+                    
+                    if (!empty($attributes)) {
+                        $friendlyExceptionAttribute = $attributes[0]->newInstance();
+                        $name = $friendlyExceptionAttribute->name . ' (' . $name . ')';
+                    }
+                }
+            } catch (\Throwable) {
```

Comment:
Where do you expect an exception? I'd remove try/catch

Reply:
You're right, the try/catch block isn't necessary here. When checking for the FriendlyException attribute, it would be better to let any exceptions propagate normally, making potential issues easier to discover. I'll remove the try/catch block. Thank you for the suggestion!